### PR TITLE
feat: [CI-4770]: Add comment dropdown for Gitlab/Bitbucket triggers

### PR DESCRIPTION
**Pull request by rutvijmehta-harness**
##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

File: src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx [hunk 1]
```diff
@@ -34,7 +36,7 @@ export const getAutoAbortDescription = ({
     return getString('triggers.triggerConfigurationPanel.autoAbortPush')
   } else if (event === eventTypes.PULL_REQUEST || event === eventTypes.MERGE_REQUEST) {
     return getString('triggers.triggerConfigurationPanel.autoAbortPR')
-  } else if (event === eventTypes.ISSUE_COMMENT) {
+  } else if (event === eventTypes.ISSUE_COMMENT || event === eventTypes.MR_COMMENT || event === eventTypes.PR_COMMENT) {
```
Comment (vardanbansal-harness) [inline src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx:39]: or alternatively, 
`[eventTypes.ISSUE_COMMENT, eventTypes.MR_COMMENT, eventTypes.PR_COMMENT].includes(event)`

File: src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx [hunk 2]
```diff
@@ -188,6 +190,10 @@ export const getEventLabelMap = (event: string): string => {
     return 'Merge Request'
   } else if (event === eventTypes.ISSUE_COMMENT) {
     return 'Issue Comment'
+  } else if (event == eventTypes.PR_COMMENT) {
+    return 'Pull Request Comment'
```
Comment (vardanbansal-harness) [inline src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx:194]: This should ideally go to strings.yaml file of `75-ci` module.

**Review by vardanbansal-harness — APPROVED** [review]

**Review by mtran7 — APPROVED** [review]

**Comment by rutvijmehta-harness** [comment]
retrigger titlecheck

**Review by mtran7 — APPROVED** [review]

File: src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx (hunk 1 above)
Comment (sahithikolichala) [reply src/modules/72-triggers/pages/triggers/utils/WebhookTriggerConfigPanelUtils.tsx:39]: agree with @vardanbansal-harness

**Review by sahithikolichala — DISMISSED** [review]

**Comment by rutvijmehta-harness** [comment]
Only sorting for Gitlab and Bitbucket to avoid regression for other SCM providers

**Comment by mtran7** [comment]
trigger cypress

**Review by mtran7 — APPROVED** [review]

**Review by sahithikolichala — APPROVED** [review]

**Pull request merged**